solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java

```diff
@@ -141,14 +146,29 @@ public void setId(int id) {
     this.id = id;
   }
 
+  public static Feature fromMap(SolrResourceLoader solrResourceLoader,
+      Map<String,Object> featureMap) {
+    final String type =
```

Comment:
changed as suggested for Feature.java here and also for the Normalizer.java class